rules/windows/defense_evasion_network_connection_from_windows_binary.toml

```diff
@@ -140,15 +140,25 @@ sequence by process.entity_id with maxspan=5m
       process.name : "iexpress.exe" or
       process.name : "installutil.exe" or
       process.name : "Microsoft.Workflow.Compiler.exe" or
-      process.name : "MSBuild.exe" or
+      (
+        process.name : "msbuild.exe" and
+          destination.ip != "127.0.0.1"
+      ) or
       process.name : "msdt.exe" or
       process.name : "mshta.exe" or
       (
-         process.name : "msiexec.exe" and not
-         dns.question.name : (
-            "ocsp.digicert.com", "ocsp.verisign.com", "ocsp.comodoca.com", "ocsp.entrust.net", "ocsp.usertrust.com",
-            "ocsp.godaddy.com", "ocsp.camerfirma.com", "ocsp.globalsign.com", "ocsp.sectigo.com", "*.local"
-         )
+        process.name : "msiexec.exe" and not
+        dns.question.name : (
+           "ocsp.digicert.com", "ocsp.verisign.com", "ocsp.comodoca.com", "ocsp.entrust.net", "ocsp.usertrust.com",
+           "ocsp.godaddy.com", "ocsp.camerfirma.com", "ocsp.globalsign.com", "ocsp.sectigo.com", "*.local"
+        ) and
+        destination.ip != "127.0.0.1" and
+        /* DigiCert IP addresses */
+        destination.ip != "192.229.211.108" and
+        destination.ip != "192.229.221.95" and
+        destination.ip != "152.195.38.76" and
+        /* Comodo CA IP addresses */
+        destination.ip != "104.18.14.101"
```

Comment:
Couldn't find a way to put them all into a single condition, if you do it the usual way it will complain that the field is not a keyword. Open to working suggestions tho

Reply:
You should test it, but potentially, using a cidrmask with a /32 block? 
`and not cidrMatch(destination.ip, "127.0.0.1/32", "192.229.211.108/32",  "192.229.221.95/32", "192.229.221.95/32", "152.195.38.76/32", "104.18.14.101/32")`
Just tried it in timelines, seems to work.

Reply:
Might also potentially make sense in that case to add `0.0.0.0`, `::1`, `127.0.0.0/8` rather than just `127.0.0.1`.